wakebox/main.cpp

```diff
@@ -98,3 +137,22 @@ int main(int argc, char *argv[])
 
 	return res;
 }
+
+char* escaped_command_env(const JAST& jast) {
```

Comment:
This messes up \\s in the arguments.
Just use: shell_escape from common/shell.h, it handles all the odd cases correctly.